ui/hooks/useAccountTotalCrossChainFiatBalance.ts

```diff
@@ -56,7 +57,12 @@ export const useAccountTotalCrossChainFiatBalance = (
     () =>
       formattedTokensWithBalancesPerChain.map((singleChainTokenBalances) => {
         const { tokensWithBalances } = singleChainTokenBalances;
+        // Attempt to use known currency symbols in map
+        // Otherwise fallback to user defined currency
         const matchedChainSymbol =
+          CHAIN_ID_TO_CURRENCY_SYMBOL_MAP[
+            singleChainTokenBalances.chainId as keyof typeof CHAIN_ID_TO_CURRENCY_SYMBOL_MAP
+          ] ??
           allNetworks[singleChainTokenBalances.chainId as `0x${string}`]
             .nativeCurrency;
         const conversionRate =
```

Comment:
This change fixes the aggregate balance issue.
Before, this relied on the user defined ticker, so was pointing to the incorrect currency rates.
Now we will try to use our hardcoded map before reaching for the user defined ticker.